### DISCUSSION TITLE
Docker support

### DISCUSSION THREAD

**Discussion by by-justin** [WebCrateApp/webcrate#52] [Ideas] [answered]
Neat project, would be even better if more deployment methods other than deta can be supported, which I tried on my raspberry pi, only to get a bunch of errors.
Offical docker support could be super helpful to noobs like me tho. Maybe considering adding a dockerfile? Huge thanks!!!

**Comment by BetaHuhn** [comment, marked as answer]
As already mentioned in other discussions (https://github.com/WebCrateApp/webcrate/discussions/42#discussioncomment-1452331, https://github.com/WebCrateApp/webcrate/discussions/26#discussioncomment-1239874), self-hosting WebCrate with or without Docker is currently not possible and will probably not be possible anytime soon for the reasons mentioned in the two linked comments.

Hope you still enjoy WebCrate using Deta Space😄
